src/command/src.jl

```diff
@@ -0,0 +1,32 @@
+function commander(c::Client, m::Message, ::Val{:source})
+	startswith(m.content, COMMAND_PREFIX * "src") || return
+		
+	regex = Regex(COMMAND_PREFIX * raw"src( help| discourse| gm| ig| j| react| src| tz)? *(.*)$")
+
+	matches = match(regex, m.content)
+
+	if matches === nothing || matches.captures[1] ∈ (" help", nothing)
+		help_commander(c, m, :source)
+	elseif lowercase(strip(matches.captures[1])) ∈ ("discourse", "gm", "ig", "j", "react", "src", "tz")
+		msg = string(BOT_REPO_URL, "/blob/main/src/command/", lowercase(strip(matches.captures[1])), ".jl")
+		reply(c, m, msg)
+	else
+		reply(c, m, "No such blob or file. Please check out $(BOT_REPO_URL) instead")
+	end
```

Comment:
Few things:

1. We should just use this constant so we don't need to maintain another command list. Just make sure that you pick the active ones only. https://github.com/Humans-of-Julia/HoJBot.jl/blob/3ec82f8b64c80a5d2dfb8e6bf472d3a87bea2014/src/main.jl#L1-L8

2. I suggest to make this more readable by factoring out the code that extracts the command:

```julia
    if matches === nothing || matches.captures[1] ∈ (" help", nothing)
        help_commander(c, m, :source)
    else
        command = lowercase(strip(matches.captures[1]))
        if command ∈ ("discourse", "gm", "ig", "j", "react", "src", "tz")
            msg = string(BOT_REPO_URL, "/blob/main/src/command/", command, ".jl")
            reply(c, m, msg)
        else
            reply(c, m, "No such blob or file. Please check out $(BOT_REPO_URL) instead")
        end
    end
```